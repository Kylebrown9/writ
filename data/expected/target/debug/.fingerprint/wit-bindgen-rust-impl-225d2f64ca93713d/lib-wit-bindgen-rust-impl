2ae35233e738eddf