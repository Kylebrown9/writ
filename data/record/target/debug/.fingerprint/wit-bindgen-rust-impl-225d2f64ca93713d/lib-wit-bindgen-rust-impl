b6525adb239eb3ff