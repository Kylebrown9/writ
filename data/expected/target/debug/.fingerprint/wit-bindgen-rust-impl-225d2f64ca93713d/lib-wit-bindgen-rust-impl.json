{"rustc":6064785002308032717,"features":"[]","target":17376663815444844849,"profile":12637318739757120569,"path":3403474873762581789,"deps":[[2346446859411652336,"syn",false,11430810127383305981],[5432826877380094970,"wit_bindgen_gen_core",false,8805643590270600382],[8442250388030655640,"wit_bindgen_gen_rust_wasm",false,15433819103424461705],[11187512571400337516,"proc_macro2",false,9475672287560845758]],"local":[{"CheckDepInfo":{"dep_info":"debug/.fingerprint/wit-bindgen-rust-impl-225d2f64ca93713d/dep-lib-wit-bindgen-rust-impl"}}],"rustflags":[],"metadata":3169560905057905687,"config":2202906307356721367,"compile_kind":0}